### PR TITLE
Meeting 4 minutes

### PR DESCRIPTION
**Pull request by simoncozens**

### CODE REVIEW

File: minutes/meeting-202001019-4.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Minutes of Meeting 4, 2020-10-17, 16:00–18:30 UTC
+
+Attendees
+
+Present
```
Comment (alerque) [inline minutes/meeting-202001019-4.md]:
```suggestion
Present (IRC)
```

OR ditch this section, we probably don't need IRC vs. Video meeting attendees separately, a merged list or only the latter is probably good.

File: minutes/meeting-202001019-4.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+liang, simon, Vlad, cjchapman, mehmetoguzderin, alerque, JohnHudson, dsinger, dscorbett, jfkthame, nmccully, Norbert
+
+* Chair: liang
```
Comment (alerque) [inline minutes/meeting-202001019-4.md]:
```suggestion
* Chair: Liang Hai
```

Either use names or IRC nicks, not both side by side in the same section. Maybe names through the attendance section, nicks for transcript?

File: minutes/meeting-202001019-4.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+## Actions
+
+* Ask W3C for a dedicated meeting space.
+* Discuss time scheduling on GitHub issue.
+* Scott, please post more notes on this.
+* Try setting up the github-bot
+
+## Attendance
+
+Vladimir Levantovsky, Liang Hai, Simon Cozens, Andrew Glass, Anthrotype, Bobby de Vos, Caleb Maclennan, Chris Chapman, David Singer, dcorbett, Duncan MacMichael, John Hudson, Johnathan Kew, Laurence, Mehmet Oguz Derin, Nat McCully, Nate Willis, Ned Holbrook, Norbert, Renzhi Li, Simon Cozens, Toshiya Suzuki, Zachary Scheuren
```
Comment (alerque) [inline minutes/meeting-202001019-4.md]:
```suggestion
Vladimir Levantovsky, Liang Hai, Simon Cozens, Andrew Glass, Anthrotype, Bobby de Vos, Caleb Maclennan, Chris Chapman, David Singer, David Corbett, Duncan MacMichael, John Hudson, Johnathan Kew, Laurence, Mehmet Oğuz Derin, Nat McCully, Nate Willis, Ned Holbrook, Norbert, Renzhi Li, Simon Cozens, Toshiya Suzuki, Zachary Scheuren
```

We aren't afraid of Unicode are we?

File: minutes/meeting-202001019-4.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+## Resolutions
+
+None
+
+## Actions
+
+* Ask W3C for a dedicated meeting space.
+* Discuss time scheduling on GitHub issue.
+* Scott, please post more notes on this.
+* Try setting up the github-bot
```
Comment (alerque) [inline minutes/meeting-202001019-4.md]:
```suggestion
* Ask W3C for a dedicated video meeting space.
* Discuss meeting time scheduling on GitHub issue.
* Research setting up IRC/github-bot.
* @scottkellum, please post more notes from implementation responsive text prototypes.
```

**Review by alerque — CHANGES_REQUESTED** [review]
Thanks for being the scribe! Looks great.


If you don't mind I might do a little more cleanup, but since I'm getting distracted I figured I should post the bits I have to start with. I can commit these later when I come back to it too.

File: minutes/meeting-202001019-4.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
+# Minutes of Meeting 4, 2020-10-17, 16:00–18:30 UTC
+
+Attendees
+
+Present (IRC)
+liang, simon, Vlad, cjchapman, mehmetoguzderin, alerque, JohnHudson, dsinger, dscorbett, jfkthame, nmccully, Norbert
+
```
Comment (lianghai) [inline minutes/meeting-202001019-4.md]:
```suggestion
```

Merge this attendance info in the IRC channel into the main section.

File: minutes/meeting-202001019-4.md [hunk 6]
```diff
@@ -0,0 +1,128 @@
...
+## Actions
+
+* Ask W3C for a dedicated video meeting space.
+* Discuss meeting time scheduling on GitHub issue.
+* Research setting up IRC/github-bot.
+* @scottkellum, please post more notes from implementation responsive text prototypes.
+
+## Attendance
+
+Vladimir Levantovsky, Liang Hai, Simon Cozens, Andrew Glass, Anthrotype, Bobby de Vos, Caleb Maclennan, Chris Chapman, David Singer, David Corbett, Duncan MacMichael, John Hudson, Johnathan Kew, Laurence, Mehmet Oğuz Derin, Nat McCully, Nate Willis, Ned Holbrook, Norbert, Renzhi Li, Simon Cozens, Toshiya Suzuki, Zachary Scheuren
```
Comment (lianghai) [inline minutes/meeting-202001019-4.md]:
```suggestion
Andrew Glass, Bobby de Vos, Caleb Maclennan\*, Chris Chapman\*, Cosimo Lupo\*, David Corbett\*, David Singer\*, Duncan MacMichael, Greg Hitchcock, John Hudson\*, Johnathan Kew\*, Laurence Penney\*, Liang Hai\*, Mehmet Oğuz Derin\*, Nat McCully\*, Nate Willis\*, Ned Holbrook\*, Norbert Lindenberg, Peter Constable\*, Renzhi Li, Scott Kellum\*, Simon Cozens\*, Toshiya Suzuki\*, Vladimir Levantovsky\*, Zachary Scheuren\*

\* Formal participants of the FTCG.

The IRC channel recorded the following nicks: liang, simon, Vlad, cjchapman, mehmetoguzderin, alerque, JohnHudson, dsinger, dscorbett, jfkthame, nmccully, Norbert
```

Changes:

- Added the following names according to my Zoom screenshots: Scott Kellum, Greg Hitchcock, Peter Constable
- Normalized the following names: Anthrotype → Cosimo Lupo, Laurence → Laurence Penney, Norbert → Lindenberg
- Removed a duplication of Simon’s name.
- Marked formal attendees.
- Applied a dumb alphabetical order.
- Incorporated the IRC attendance info.

**Review by lianghai — DISMISSED** [review]
Additional improvements to the Attendance sections.

**Comment by lianghai** [comment]
Thank you guys, especially Simon… I couldn’t process this in time despite your great effort of taking all the notes…

Now I’m trying (some weird network issue is happening) to mark this PR as “non-substantive” at https://labs.w3.org/repo-manager/pr/id/w3c/font-text-cg/26 to pass the bot’s requirement for now. @simoncozens, you should [link your GitHub account with your W3C account](https://www.w3.org/users/myprofile/connectedaccounts) though, to prevent this from happening again.

**Review by alerque — DISMISSED** [review]

**Comment by simoncozens** [comment]
I've linked my accounts.

File: minutes/meeting-202001019-4.md [hunk 7]
```diff
@@ -0,0 +1,127 @@
+# Minutes of Meeting 4, 2020-10-17, 16:00–18:30 UTC
```
Comment (lianghai) [inline minutes/meeting-202001019-4.md]:
```suggestion
# Minutes of Meeting 4, 2020-10-19, 16:00–18:30 UTC
```

Corrected the date.

**Review by alerque — APPROVED** [review]

**Pull request merged**